census/src/main/java/io/grpc/census/CensusTracingModule.java

```diff
@@ -211,17 +215,19 @@ private static EndSpanOptions createEndSpanOptions(
         .build();
   }
 
-  private static void recordMessageEvent(
+  private void recordMessageEvent(
       Span span, MessageEvent.Type type,
       int seqNo, long optionalWireSize, long optionalUncompressedSize) {
-    MessageEvent.Builder eventBuilder = MessageEvent.builder(type, seqNo);
-    if (optionalUncompressedSize != -1) {
-      eventBuilder.setUncompressedMessageSize(optionalUncompressedSize);
-    }
-    if (optionalWireSize != -1) {
-      eventBuilder.setCompressedMessageSize(optionalWireSize);
+    if (addMessageEvents) {
```

Comment:
Could you invert this to reduce indentation and make it a bit easier to read?

```java
if (!addMessageEvents) {
  return;
}
```

Reply:
ok